### PR TITLE
Add vcs and rosdep as requirements

### PR DESCRIPTION
**Pull request by ojasjoshi**
*Issue #, if available:*

*Description of changes:*
Update the README to have vcs and rosdep as requirements.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ _RoboMaker sample applications include third-party software licensed under open-
 
 - [ROS Kinetic](http://wiki.ros.org/kinetic/Installation/Ubuntu) / [ROS Melodic](http://wiki.ros.org/melodic/Installation/Ubuntu) - Other versions may work, however they have not been tested
 - [Colcon](https://colcon.readthedocs.io/en/released/user/installation.html) - Used for building and bundling the application.
+- [vcs](https://github.com/dirk-thomas/vcstool#how-to-install-vcstool) - Used to pull in sample app dependencies
+- [rosdep](http://wiki.ros.org/rosdep) - rosdep is a command-line tool for installing system dependencies.
```
Comment (ahtsan) [inline README.md]: nitpick - Do you want to point to the exact section for installation? like http://wiki.ros.org/rosdep#Installing_rosdep, Just to be consistent with all the reference in this section

File: README.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ _RoboMaker sample applications include third-party software licensed under open-
 
 - [ROS Kinetic](http://wiki.ros.org/kinetic/Installation/Ubuntu) / [ROS Melodic](http://wiki.ros.org/melodic/Installation/Ubuntu) - Other versions may work, however they have not been tested
 - [Colcon](https://colcon.readthedocs.io/en/released/user/installation.html) - Used for building and bundling the application.
+- [vcs](https://github.com/dirk-thomas/vcstool#how-to-install-vcstool) - Used to pull in sample app dependencies
```
Comment (mm318) [inline README.md]: I think this should say `vcstool` (software name) instead of `vcs` (executable name).

File: README.md (hunk 1 above)
Comment (mm318) [inline README.md]: I would say "system dependencies of ROS packages".

File: README.md (hunk 2 above)
Comment (mm318) [reply README.md]: I would add "sample app dependencies that are only available from source, not from `apt` or `pip`".

**Review by mm318 — APPROVED** [review]

**Pull request merged**